src/main/java/com/github/cchacin/cucumber/steps/RestSteps.java

```diff
@@ -178,4 +179,10 @@ public void response_json_path_element_should_be(final String jsonPath, final St
         final Object responseValue = JsonPath.read(this.responseValue, jsonPath);
         assertThat(String.valueOf(responseValue)).isEqualTo(value);
     }
+
+    @Then("^response json path list \"(.*?)\" should be of length (.*?)$")
```

Comment:
use regular expression for digits and updated version https://github.com/cchacin/cucumber-common-steps/commit/66f9b2ddd13fc1fb324d837ff776bbeca512172c